src/rules/utils.ts

```diff
@@ -648,6 +648,16 @@ export type FunctionExpression =
   | TSESTree.ArrowFunctionExpression
   | TSESTree.FunctionExpression;
 
+export const isCallExpression = (
```

Comment:
We don't need this utility function, as you can just do `node?.type === AST_NODE_TYPES.CallExpression`